File: solutions/cpp/nsum.cpp
// https://open.kattis.com/problems/nsum
// Difficulty: 1.3 (Easy)

#include <iostream>
using namespace std;

int main(){
    int len;
    cin >> len;

    int sum = 0;
    for(int i = 0; i < len; i++)
    {
        int x;
        cin >> x;
        sum += x;
    }

    cout << sum;

    return 0;
}

File: solutions/java/nsum.java
// https://open.kattis.com/problems/fyi
// Difficulty: 1.3 (Easy)

import java.util.Scanner;

public class nsum
{
    public static void main(String[] args)
    {
        Scanner scn = new Scanner(System.in);
        Integer.parseInt(scn.nextLine());

        int sum = 0;
        
        String[] numberLine = scn.nextLine().split(" ");

        for (int i = 0; i < numberLine.length; i++)
        {
            sum += Integer.parseInt(numberLine[i]);
        }

        System.out.println(sum);

        scn.close();
    }
}